src/coreclr/vm/methodtablebuilder.cpp

```diff
@@ -10051,6 +10051,10 @@ void MethodTableBuilder::CheckForSystemTypes()
                     // The Procedure Call Standard for ARM 64-bit (with SVE support) defaults to
                     // 16-byte alignment for __m256.
 
+                    pLayout->m_ManagedLargestAlignmentRequirementOfAllMembers = 16;
```

Comment:
Was the unintentional?

Reply:
In GitHub, line 10054 is matched to line 10058. And it marks line 10054 as newly added.